runtime/jvmti/jvmtiStackFrame.cpp

```diff
@@ -883,3 +883,16 @@ jvmtiInternalGetStackTrace(
 }
 
 } /* extern "C" */
+
+#if JAVA_SPEC_VERSION >= 25
+jvmtiError JNICALL
+jvmtiClearAllFramePops(jvmtiEnv *env,
+	jthread thread)
```

Comment:
Suggest this should not be split.
The definition should be within the `extern "C"` block (i.e. before line 885).

Reply:
Updated.

Reply:
I think it only makes the code harder to read when short function signatures are split across multiple lines. That might make sense for functions with many parameters, but I don't think "two" qualifies as "many". Please put the declaration in `jvmti_internal.h` and here on one line:
```c
jvmtiClearAllFramePops(jvmtiEnv *env, jthread thread);
```

Reply:
Updated.